code/game/objects/structures/crates_lockers/largecrate.dm

```diff
@@ -71,6 +72,15 @@
 	if(power >= EXPLOSION_THRESHOLD_VLOW)
 		unpack()
 
+/obj/structure/largecrate/proc/take_damage(damage)
+	health -= damage
+	if(health <= 0)
+		unpack()
+
+/obj/structure/largecrate/bullet_act(obj/item/projectile/P)
+	take_damage(P.damage*0.5)
```

Comment:
You should use `/obj/item/projectile/calculate_damage()` rather than `P.damage` directly so it accounts for falloff/buildup

Reply:
changed in 23f031622b398a028d7c647be2216e30a361b959